src/ftest.jl

```diff
@@ -127,6 +155,12 @@ function ftest(mods::LinearModel...; atol::Real=0.0)
     return FTestResult(Int(nobs(mods[1])), SSR, df, r2.(mods), fstat, pval)
 end
 
+function show(io::IO, fstat::SingleFTestResult)
+    print(io, "F-statistic: ", round(fstat.f, sigdigits=4), " ")
```

Comment:
Would it make sense to use `StatsBase.TestStatistic` instead of `@printf` here?

Reply:
Well I think I added that type in StatsBase only because it was needed to get two decimals while still using the matrix printing code. Other than that it doesn't have much benefit, does it?

That said, maybe we should print only 2 decimal digits rather than 4 for consistency.

Reply:
Consistency is why I suggested it: if all test statistics use `TestStatistic`, then they'll all be printed the same.

Reply:
Yeah, makes sense.

Reply:
Interesting discussion! I made it `    print(io, "F-test against the null model:\nF-statistic: ", TestStat(fstat.f), " ")`